corehq/apps/accounting/models.py

```diff
@@ -1684,10 +1684,45 @@ def transfer_credits(self, subscription=None):
             )
 
     def send_renewal_reminder_email(self):
-        ...
+        template = 'accounting/email/subscription_renewal_reminder.html'
+        template_plaintext = 'accounting/email/subscription_renewal_reminder.txt'
+        self._send_ending_reminder_email(template, template_plaintext)
 
     def send_subscription_ending_email(self):
-        ...
+        template = 'accounting/email/subscription_ending.html'
+        template_plaintext = 'accounting/email/subscription_ending.txt'
+        self._send_ending_reminder_email(template, template_plaintext)
+
+    def _send_ending_reminder_email(self, template, template_plaintext):
+        today = datetime.date.today()
+        num_days_left = (self.date_end - today).days
```

Comment:
Nit: It looks like we only use this when logging, so it probably isn't severe, but...the day when this reminder actually gets sent out, vs the day that we fetched this subscription may not align. I.e. 1 minute before midnight, we might fetch all subscriptions set to expire in 10 days. If the iteration took long enough, by the time we process a subscription, it might be the next day, and then we'd log that we sent a "9 day subscription reminder", which isn't one of our defined periods. Could we pass the actual date period instead?